.github/workflows/validate.yml

```diff
@@ -169,12 +148,21 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    - name: install deps
-      run: |
-        sudo apt -qq update
-        sudo apt -qq install gperf
+      # We have to run this under Docker as Ubuntu (host) does not support all
+      # the architectures we want to compile test against, and Dockerfile uses
+      # Debian (which does).
+      #
+      # XXX: as currently this is the only job that is using Docker, we are
+      # building and using the runcimage locally. In case more jobs running
+      # under Docker will emerge, it will be good to have a separate make
+      # runcimage job and share its result (the docker image) with whoever
+      # needs it.
+    - uses: satackey/action-docker-layer-caching@v0.0.11
+      continue-on-error: true
+    - name: build docker image
+      run: make runcimage
     - name: make release
-      run: make release
+      run: make RELEASE_ARGS="-a arm64 -a armel -a armhf -a ppc64le" release
```

Comment:
It would be nice if there was a `make releaseall` or something target that built for every architecture we support.

Reply:
done